.travis.yml

```diff
@@ -100,7 +100,8 @@ matrix:
           dist: trusty
           sudo: required
           compiler: gcc
-          node_js: "6"
```

Comment:
It definitely needs version 6 though , given `Buffer.form(..., 'hex')` is only supported by that.